### ISSUE TITLE
Apply to RGB pictures

### ISSUE THREAD

**Issue by giovanni-turra** [jocicmarko/ultrasound-nerve-segmentation#39]
I would apply U-Net to RGB pictures (input). Could I understand if is it possible with a small dataset? Is it possible with U-Net and this implementation?

**Comment by jocicmarko** [comment]
Yes, it is possible, just make sure you specify that input has 3 channels:
```
inputs = Input((img_rows, img_cols, 3))
```

**Comment by giovanni-turra** [comment]
Thanks Marko.

But, from a structure point of view, U-Net apply an averange across color axis after input step?

Or is it able to combine information from single channel to increase segmentation phase?

I ask you this because from U-Net paper I think this part it is not covered.

**Comment by titusaj** [comment]
If changing over to color image segmentation I took the approach where the train_mask images are also converted to three channel images for ease of input into the model.

However I get reshape/incompatible shape error when run. 

The model runs fine at the input level (ie. accepts images into model).

How would the model itself have to change to allow for three channel input/convolution.

**Comment by namsu-kim-unsw** [comment]
I tried making mine RGB and ran it but it outputs a completely black picture

Edit: I think the reason for this was because of the number of epochs - it was too low.
With a higher number of epochs, it did generate an image but the dice is super low :( around 5%

**Comment by matsujju** [comment]
@jocicmarko Where to input images and mask in the data.py file if I have image data outside kaggle data?
